### ISSUE TITLE
Add log initializer to stratis_dumpmetadata program

### ISSUE THREAD

**Issue by bgurney-rh** [stratis-storage/stratisd#2518]
We should add a log initializer to the stratis_dumpmetadata program, similar to the one in stratisd (an example of which is listed below), which is necessary since stratis_dumpmetadata is a program built separately from straitsd.

(example from `src/bin/stratisd.rs` from develop-2.3.0 as of now:)

```
/// Configure and initialize the logger.
/// If optional log_level argument is set, use that to set the log level
/// for both stratisd and libstratis. Otherwise, read log configuration
/// parameters from the environment if RUST_LOG is set. Otherwise, just
/// accept the default configuration, which is to log at the severity of
/// error only.
fn initialize_log(log_level: Option<&str>) {
    let mut builder = Builder::new();

    if let Some(log_level) = log_level {
        builder.filter(
            Some("stratisd"),
            LevelFilter::from_str(log_level)
                .expect("argument parser only accepts valid log levels"),
        );
        builder.filter(
            Some("libstratis"),
            LevelFilter::from_str(log_level)
                .expect("argument parser only accepts valid log levels"),
        );
    } else if let Ok(s) = env::var("RUST_LOG") {
        builder.parse_filters(&s);
    }

    builder.init()
}
```

**Comment by mulkieran** [comment]
I think we could do this immediately. Given that this is a troubleshooting script anyway, I think it kind of makes  sense to set the log level at trace, and not bother with command-line args or reading from the environment.

**Comment by mulkieran** [comment]
Closed by #2526. We're going at this time with using RUST_LOG env variable to set the log level. We may decide later to add  a command-line arg, but we may decide it is unnecessary.
